scipy/sparse/linalg/tests/test_expm_multiply.py

```diff
@@ -260,19 +261,28 @@ def test_sparse_expm_multiply_interval_dtypes(self):
         A = scipy.sparse.diags_array(np.arange(5),format='csr', dtype=int)
         B = np.ones(5, dtype=int)
         Aexpm = scipy.sparse.diags_array(np.exp(np.arange(5)),format='csr')
+        BI = np.identity(5, dtype=int)
+        BI_sparse = scipy.sparse.csr_array(BI)
         assert_allclose(expm_multiply(A,B,0,1)[-1], Aexpm.dot(B))
+        assert_allclose(np.diag(expm_multiply(A,BI_sparse,0,1)[-1]), Aexpm.dot(B))
```

Comment:
Please can you fix the formatting in the lines added in this file to follow PEP8, ie.
```suggestion
        assert_allclose(np.diag(expm_multiply(A, BI_sparse, 0, 1)[-1]), Aexpm.dot(B))
```